src/pages/trees/[treeid].js

```diff
@@ -385,13 +389,30 @@ export default function Tree({
             icon={<img src={historyIcon} alt="age" />}
           />
         )}
+        {tree.species && (
+          <TreeTag
+            TreeTagValue={tree.species}
+            title="Natural Origin"
+            icon={<img src={originIcon} alt="origin" />}
+          />
+        )}
+
         {tree.gps_accuracy && (
           <TreeTag
             TreeTagValue={tree.gps_accuracy}
             title="GPS Accuracy"
             icon={<img src={accuracyIcon} alt="accuracy" />}
           />
         )}
+
+        {tree.morphology && (
+          <TreeTag
+            TreeTagValue={tree.morphology}
+            title="Diameter at Breast Height"
+            icon={<img src={diameterIcon} alt="diameter" />}
+          />
+        )}
+
```

Comment:
@pierrelstan we are unnecessarily making things complicated.

Why don't you just put it here? 

```
<TreeTag
  value={tree.morphology + " cm"}
  title="Diameter at Breast Height"
/>
```

Reply:
Yours is much simpler, thank you for the eye-opening.